### ISSUE TITLE
Do more in a single osascript call

### ISSUE THREAD

**Issue by jgwhite** [withassociates/dj54b-api-phoenix#3]
Right now we’re issuing a separate System.cmd for each property (volume, state, name, artist, id). Time to learn enough AppleScript to retrieve all those details in one go.

**Comment by jgwhite** [comment]
``` sh
$ osascript -e 'tell application "Spotify" to "" & (get sound volume) & "|" & (get player state) & "|" & (get the spotify url of the current track) & "|" & (get the name of the current track) & "|" & (get the artist of the current track)'
30|paused|spotify:track:1cTIkQUcWtwooj2rDcNB5n|My Way|Limp Bizkit
```

**Comment by jgwhite** [comment]
Closed by #4
